apps/bare-expo/MainNavigator.tsx

```diff
@@ -34,12 +34,11 @@ export function optionalRequire(requirer: () => { default: React.ComponentType }
   }
 }
 const routes: RoutesConfig = {
-  [testSuiteRouteName]: TestSuite,
+  [testSuiteRouteName]: TestStackNavigator,
 };
 
-// We'd like to get rid of `native-component-list` being a part of the final bundle.
-// Otherwise, some tests may fail due to timeouts (bundling takes significantly more time).
-// See `babel.config.js` and `moduleResolvers/nullResolver.js` for more details.
+// TODO vonovak there's potential for skipping the require of APIs tab as it's not used in CI
+// could use metro config to exclude it from bundling
```

Comment:
it's unclear to me for this TODO. i was thinking that's your intention to keep the whole NCL chunks in the bundle so you removed the babel nullResolver and can future test NCL in CI?

Reply:
@kudo there are 3 tabs in bare expo: tests, apis, and components.

The APIs tab is not being used by e2e tests, so I'm just saying that it doesn't have to be resolved. I don't know if that would help or if it's possible. I'd leave it for later.